VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -50,6 +50,12 @@ public static void setAuthorAttributionText(@NonNull TextView textView,
             final CharSequence formattedTime = getRelativeTimeSpanString(context,
                     initialTimeStampMs, authorData.getCreatedAt().getTime());
             final String authorLabel = authorData.getAuthorLabel();
+            final StringBuilder authorLabelFormatted = new StringBuilder();
+            if (authorLabel != null) {
+                authorLabelFormatted.append("(");
+                authorLabelFormatted.append(authorLabel);
+                authorLabelFormatted.append(")");
```

Comment:
Yeah, we should localize the parentheses. In general it's better to not make any assumptions that anything about language is universal. Compare how some languages use << and  >> instead of "" for quotations.